stable/redis-ha/templates/redis-haproxy-service.yaml

```diff
@@ -29,8 +29,8 @@ spec:
     targetPort: {{ .Values.haproxy.readOnly.port }}
 {{- end }}
 {{- if .Values.exporter.enabled }}
-  - name: exporter-port
-    port: {{ .Values.haproxy.exporter.port }}
+  - name: {{ .Values.metrics.portName }}
```

Comment:
Is it necessary to template the port name?

Reply:
@ssalaues There's a quite common situation when users have already defined format for all exporter services port name and it will be nice to have an opportunity to have such declaration in chart instead of creating scrape configurations for every uncommon naming like `exporter`, `exporter-port`, `prom-exporter`, etc.